index.html

```diff
@@ -15,7 +15,7 @@
     <meta name="generator" content="Prairie Creek Game">
     <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
 
-    <title>My App</title>
+    <title>Spirits at Prairie Creek Park</title>
     <meta name="description" content="My App description">
```

Comment:
Right under the title, we have a meta description tag that still just calls it "My App". Let's bring all this boilerplate code up to snuff so we don't have conflicting ideas in the source code.